jasmine/SpecRunner.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Jasmine Spec Runner v3.4.0</title>
+
+  <link rel="shortcut icon" type="image/png" href="lib/jasmine-3.4.0/jasmine_favicon.png">
+  <link rel="stylesheet" href="lib/jasmine-3.4.0/jasmine.css">
+
+  <script src="lib/jasmine-3.4.0/jasmine.js"></script>
+  <script src="lib/jasmine-3.4.0/jasmine-html.js"></script>
+  <script src="lib/jasmine-3.4.0/boot.js"></script>
+
+  <!-- include source files here... -->
+  <script src="../CoffeeShop_unit_test.js"></script>
+
+  <!-- include spec files here... -->
+  <script src="spec/CoffeeShop_unit_test.js"></script>
```

Comment:
Every language has now at least one package manager. JS's most comonly used is https://www.npmjs.com/

To avoid adding library files to the main repository and make it more easy to update to newer versions.